.github/workflows/test.yml

```diff
@@ -7,6 +7,9 @@ on:
 
 jobs:
   linux:
+    env:
```

Comment:
Should credentials be exposed to the `windows` workflow as well?

Reply:
For some reason Windows fails to connect with SauceLabs with an [error](https://github.com/karma-runner/integration-tests/runs/5983858482?check_suite_focus=true) referring to a proxy or firewall configuration. I do not have the time to debug that for now, but I am opening an issue to keep track of this missing environment in our CI.